### PR TITLE
feat(Rows): images with aspect ratio in rows

### PR DESCRIPTION
**Pull request by pladaria**

### CODE REVIEW

**Review by yceballost — APPROVED** [review]
p e r f e c t !

**Review by AnaMontes11 — APPROVED** [review]

File: src/list.tsx [hunk 1]
```diff
@@ -173,18 +170,12 @@ const Content: React.FC<ContentProps> = ({
 
     return (
         <Box paddingY={16} className={classes.content}>
```
Comment (anabelengp) [inline src/list.tsx:172]: I think that in figma they have specified a special alignment for the items in the row, depending on how many items are displayed in rowBody:
**from Figma:**
image

**Our result:**
image

Should we review this with @yceballost ??

**Review by anabelengp — APPROVED** [review]

File: src/list.tsx (hunk 1 above)
Comment (pladaria) [reply src/list.tsx:172]: image
It was an error in spec definition. Specs have been updated

Comment (anabelengp) [reply src/list.tsx:172]: solved: fixed in specifications

**Pull request merged**